patterns/1-initial/principles.md

```diff
@@ -0,0 +1,152 @@
+* To increase success of new hires.
+* To build actionable documentation.
+
+### Provide clear InnerSource principles
+
+Once teams understand which problems InnerSource will help them address, the next
+step is to explain which principles help address these challenges. 
+
+Based on basic open source development principles the following guidelines 
+have been proven successful:
+
+1. Code must be transparently hosted within the organisation
+
+Source code, documentation, data relevant for project development must be
+available and easy to find for anyone in the organisation.
+
+2. Contributions over feature requests
+
+All stakeholders of a project act as potential contributors and are being
+treated and supported as such. Contributions remain suggestions instead of
+requirements. Coordination before a contribution helps avoid wasted effort.
+Projects provide contribution guidelines to avoid friction.
+
+3. Mistakes are opportunities for learning
+
+With work visible across the entire organisation any mistake is visible to
+everyone. As a result a culture must be established in which mistakes are
+opportunities for learning instead of failure that should be avoided at all
+cost.
+
+4. Written over verbal communication
+
+For projects that span multiple teams, potentially on diverging meeting
+schedules, it needs to be possible to collaborate asynchronously. The goal for
+InnerSource projects is to recruit new contributors. For that, potential future
+contributors need to be able to follow the project progress on a self serve
+basis with a very low barrier to entry. If project relevant communication
+happens through synchronous communication, arguments discussed need to be made
+transparent in the written channel - decisions should be finalized only there.
+As a side effect this leads to passive base documentation that is very valuable
+for any newcomer to the project.
+
+5. Allow written advise to accumulate in a persistent, searchable archive
+
+All project communication, in particular decisions taken and discussions leading
+up to those decisions need to be archived. It must be possible to reference
+communication via stable URLs. Previous communication needs to be stored in a
+way that can easily be searched.
+
+Two caveats though:
+
+1. This does not replace structured documentation. It can serve as a starting point
+to collect structured documentation though.
+
+2. There are exceptions to the rule of everything being written and accessible to
+the entire organisation: People related discussions as well as security related
+discussions are sensitive and should not be held in public.
+
+6. Reward Trusted Committership
+
+All contributions (e.g. source code, documentation, bug reports, input to
+discussions, user support, marketing) are welcome and are being rewarded. 
+Those showing support for the project are being invited as
+[Trusted Committers](../2-structured/trusted-committer.md). All Trusted Committers of a project are published.
```

Comment:
```suggestion
(1) Code must be transparently hosted within the organisation

Source code, documentation, data relevant for project development must be
available and easy to find for anyone in the organisation.

(2) Contributions over feature requests

All stakeholders of a project act as potential contributors and are being
treated and supported as such. Contributions remain suggestions instead of
requirements. Coordination before a contribution helps avoid wasted effort.
Projects provide contribution guidelines to avoid friction.

(3) Mistakes are opportunities for learning

With work visible across the entire organisation any mistake is visible to
everyone. As a result a culture must be established in which mistakes are
opportunities for learning instead of failure that should be avoided at all
cost.

(4) Written over verbal communication

For projects that span multiple teams, potentially on diverging meeting
schedules, it needs to be possible to collaborate asynchronously. The goal for
InnerSource projects is to recruit new contributors. For that, potential future
contributors need to be able to follow the project progress on a self serve
basis with a very low barrier to entry. If project relevant communication
happens through synchronous communication, arguments discussed need to be made
transparent in the written channel - decisions should be finalized only there.
As a side effect this leads to passive base documentation that is very valuable
for any newcomer to the project.

(5) Allow written advise to accumulate in a persistent, searchable archive

All project communication, in particular decisions taken and discussions leading
up to those decisions need to be archived. It must be possible to reference
communication via stable URLs. Previous communication needs to be stored in a
way that can easily be searched.

Two caveats though:

1. This does not replace structured documentation. It can serve as a starting point
to collect structured documentation though.

2. There are exceptions to the rule of everything being written and accessible to
the entire organisation: People related discussions as well as security related
discussions are sensitive and should not be held in public.

(6) Reward Trusted Committership

All contributions (e.g. source code, documentation, bug reports, input to
discussions, user support, marketing) are welcome and are being rewarded. 
Those showing support for the project are being invited as
[Trusted Committers](../2-structured/trusted-committer.md). All Trusted Committers of a project are published.
```

Markdown was trying to render these numbers as a numbered-list, which ended up looking odd.
This fixes that.